src/scenes/guis/end_game.gd

```diff
@@ -0,0 +1,9 @@
+extends Control
+
+var parent:Node
```

Comment:
I'm not sure what `parent` is doing. It doesn't seem to be used in the code, and it doesn't provide any better typing over `get_parent()`. Is it needed?

Reply:
Nope, I had some silly idea and forgot to get rid of it.